spec/swagger_helper.rb

```diff
@@ -6,20 +6,20 @@
   # Specify a root folder where Swagger JSON files are generated
   # NOTE: If you're using the rswag-api to serve API descriptions, you'll need
   # to ensure that it's configured to serve Swagger from the same folder
-  config.swagger_root = Rails.root.join("swagger").to_s
+  config.openapi_root = Rails.root.join("swagger").to_s
```

Comment:
Just to check: is this about these deprecation warnings? https://github.com/rswag/rswag/pull/702

Reply:
ah yes, I probably should have put those in a different commit but I ended up fixing them as I noticed them